lib/plugins/wallet/bitgo/bitgo.js

```diff
@@ -36,21 +38,22 @@ function sendCoins (account, address, cryptoAtoms, cryptoCode) {
         amount: cryptoAtoms.toNumber(),
         walletPassphrase: account.walletPassphrase
       }
-      return wallet.sendCoins(params)
+      return wallet.send(params)
     })
     .then(result => {
       return result.hash
     })
     .catch(err => {
-      if (err.message === 'Insufficient funds') throw new E.InsufficientFundsError()
+      if (err.message === 'insufficient funds') throw new E.InsufficientFundsError()
       throw err
     })
 }
 
 function balance (account, cryptoCode) {
   return checkCryptoCode(cryptoCode)
     .then(() => getWallet(account))
-    .then(wallet => BN(wallet.wallet.spendableConfirmedBalance))
+    // TODO This was previously spendableConfirmedBalance, what happened to it on v2?
```

Comment:
Shoud I use the minimum value between confirmedBalance and spendableBalance? Not sure what spendableConfirmedBalance means, and it not really documented on their api.

Reply:
Some tests:

```
0.0013 cashout =>
balance: 0.09025480
confirmedBalance: 0.01075465
spendableBalance: 0.09025480

0.0013 cashout =>
balance: 0.08891092
confirmedBalance: 0
spendableBalance: 0.08891092

0.0005 cashin =>
balance: 0.08945823
confirmedBalance: 0
spendableBalance: 0.08891092
```

Reply:
I'm converting before printout for readability.